packages/bot-web-ui/src/pages/server-side-bot/add-bot/inputs/qs-input.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { MouseEvent } from 'react';
+import classNames from 'classnames';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { Input, Popover } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { useDBotStore } from 'Stores/useDBotStore';
+
+type TQSInput = {
+    name: string;
+    onChange: (key: string, value: string | number | boolean) => void;
+    type?: string;
+    attached?: boolean;
+    should_have?: { key: string; value: string | number | boolean }[];
+    disabled?: boolean;
+    min?: number;
+    max?: number;
+    has_currency_unit?: boolean;
+};
+
+const QSInput: React.FC<TQSInput> = observer(
+    ({
+        name,
+        onChange,
+        type = 'text',
+        attached = false,
+        disabled = false,
+        min,
+        max,
+        has_currency_unit = false,
+    }: TQSInput) => {
+        const {
+            ui: { is_desktop },
+            client: { currency },
+        } = useStore();
+        const { quick_strategy } = useDBotStore();
+        const { loss_threshold_warning_data } = quick_strategy;
+
+        const [has_focus, setFocus] = React.useState(false);
+        const { setFieldValue, setFieldTouched } = useFormikContext();
+        const is_number = type === 'number';
+        const max_value = 999999999999;
```

Comment:
value seems sus. is this a max integer check? or is it an actual BE limitation - could use. if just an max integer check u can use  `Number.MAX_SAFE_INTEGER`

Reply:
This is done ✅